app/drivers/tools/repair/java/ET.py

```diff
@@ -0,0 +1,147 @@
+import os
+import json
+from pathlib import Path
+
+from app.drivers.tools.repair.AbstractRepairTool import AbstractRepairTool
+
+
+class ET(AbstractRepairTool):
+    def __init__(self):
+        self.name = os.path.basename(__file__)[:-3].lower()
+        super().__init__(self.name)
+        #self.image_name = "et-dev"
+        self.image_name = "xmcp/et:231004.2"
+        self.hash_digest = "sha256:68d39f183fa7c5bbba4e55d3150f9e1ae1ec027c6034eee18801dd6b5d04b6ec"
```

Comment:
Hi... Just a small thing in relation to this digest, earlier the code in Cerberus was incorrect and was using the local ID of the image. This has been fixed, and I was looking at some of the images. Just to gently inform about this fix,  sorry :worried:

Reply:
@xmcp can you help with this last detail, and help to close this PR?

Reply:
Okay, I will update the digest